firmware/development/kampela/Readme.md

```diff
@@ -0,0 +1,34 @@
+Wee are trying to build:
```

Comment:
Add markdowned structure for sections, like `# Building`, `## Requirements`. Also a few words on what this is.